### PR TITLE
ability to setup host using own postgress and ability to set custom storageclass and size

### PR DESCRIPTION
**Pull request by strelok899**

### CODE REVIEW

**Comment by strelok899** [comment]
@nushkovg added some more fixes and abilities

**Comment by strelok899** [comment]
@nushkovg would be glad if you can take a look here

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
+    "githubPullRequests.ignoredPullRequestBranches": [
+        "main"
+    ]
+}
```
Comment (nushkovg) [inline .vscode/settings.json:5]: Is this required for something?

File: charts/outline/templates/001-deployment.yaml [hunk 2]
```diff
@@ -16,6 +16,23 @@ spec:
         app: {{ .Release.Name }}
         release: {{ .Release.Name }}
     spec:
+      securityContext:
+        fsGroup: 1001
+        runAsUser: 1001
+        runAsGroup: 1001
```
Comment (nushkovg) [inline charts/outline/templates/001-deployment.yaml:22]: Does the deployment face any issues without these permissions?

File: charts/outline/values.yaml [hunk 3]
```diff
@@ -39,6 +39,8 @@ replicaCount: 1
 ## @param outline.service.url The URL where the application will be accessible.
 ## @param outline.service.port The port on which the application will run.
 ## @param outline.fileStorage.type Type of file storage to be used (e.g., local, s3).
+## @param outline.fileStorage.storageClassName The storageclass name to use default "".
+## @param outline.fileStorage.storageSize The storageclass size to use default 250Gi.
```
Comment (nushkovg) [inline charts/outline/values.yaml:43]: Did you use the `bitnami-helm-readme-generator` for this?

**Review by nushkovg — COMMENTED** [review]
Hey @strelok899 I have left some comments.

File: .vscode/settings.json (hunk 1 above)
Comment (strelok899) [reply .vscode/settings.json:5]: oh its from vs code i will delete

File: charts/outline/templates/001-deployment.yaml (hunk 2 above)
Comment (strelok899) [reply charts/outline/templates/001-deployment.yaml:22]: yes no permissions to write to local storage

File: charts/outline/values.yaml (hunk 3 above)
Comment (strelok899) [reply charts/outline/values.yaml:43]: no with hands its 2 rows

File: charts/outline/templates/001-deployment.yaml (hunk 2 above)
Comment (nushkovg) [reply charts/outline/templates/001-deployment.yaml:22]: Can we make this optional? Only when it's local storage. The other stuff looks good.

**Review by nushkovg — CHANGES_REQUESTED** [review]

File: charts/outline/templates/001-deployment.yaml (hunk 2 above)
Comment (strelok899) [reply charts/outline/templates/001-deployment.yaml:23]: @nushkovg done

**Review by nushkovg — APPROVED** [review]

**Pull request merged**